### ISSUE TITLE
Internet Health icon footer

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#69]
Create an icon for "Internet Health" to be placed in the network site footer
<img width="1053" alt="screen shot 2017-02-08 at 10 07 22 am" src="https://cloud.githubusercontent.com/assets/22157921/22750644/75b9ebaa-ede6-11e6-88cc-181538bf5ed3.png">

**Comment by xmatthewx** [comment]
@samanthaburton – Hi Sam. We'd like to add a link in our footer to the internet health page on Mozilla.org. Doing that requires an icon. Yikes. Any suggestions? Has this been discussed before? An apple? A cross? A lightning bolt? ⚡️ 

@taisdesouzalessa @gvn  – we can launch without this and then add it when ready. Plenty of other things on our plate. :plate_with_cutlery: 😃

**Comment by gvn** [comment]
Yeah, I can just comment it out for now.

**Comment by samanthaburton** [comment]
Hello! Yay for icons!

So we don't have one yet - closest thing we've been using is a kind of heart for the Internet Health Report, developed by MoCo (a la the attached example).

I think it'd be a good idea to collaborate with the MoCo design team on this (namely Tim Murray and Yuliya Gorlovetsky) to land on something we can use across all Mozilla properties (as I'm sure they'll also link to the moz.org internet health page!) - but I think you could put a placeholder for now and do that in phase 2, if need be. I'm also happy to help facilitate moving a collab on this forward, if that would be helpful!

![unknown](https://cloud.githubusercontent.com/assets/16709293/22803271/41ab708e-eee2-11e6-9e1a-9c7e4410f711.png)

**Comment by xmatthewx** [comment]
We use a heart for donate. I'll send a note to Yuliya.

**Comment by hannahkane** [comment]
@xmatthewx - do you have an update on this? Is there anything I can do to help unblock?

**Comment by sabrinang** [comment]
Matthew forwarded the icon Yulia's team has been using for Internet Health to me and I made a fill version of the line icon but it doesn't scale well for footer use (I tried various stroke widths):
<img width="1002" alt="screen shot 2017-03-06 at 1 34 18 pm" src="https://cloud.githubusercontent.com/assets/2492510/23630930/b2d2ce7c-0271-11e7-9b39-b2e4ec6b1e70.png">

I haven't come up with an alternative solution yet but can revisit this issue

**Comment by gvn** [comment]
Is this an icon that should go into all our sites? If so we should do this work in `mofo-bootstrap`.

**Comment by xmatthewx** [comment]
We have a lot of critical work on our plate. Let's pause this and other non critical things until that stuff is feeling like no big deal.

**Comment by hannahkane** [comment]
Sorry to distract. Moving to upcoming heartbeat.

**Comment by kristinashu** [comment]
We will be adding Internet Health as a main level item so a link is no longer needed in the footer.
